dashboard/src/components/Table/TreeTable.tsx

```diff
@@ -10,6 +10,8 @@ import { TableRow, TableCell } from '../ui/table';
 
 import { TreeTableBody, zOrigin } from '../../types/tree/Tree';
 
+import ColoredCircle from '../ColoredCircle/ColoredCircle';
```

Comment:
please don't use `../` we still don't added the lint rule but avoid it as much as possible